src/components/RepoItem/RepoItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './RepoItem.module.scss';
+import classNames from 'classnames/bind';
+import { Repo } from '../../modules/repo';
+import starImg from '../../images/star.png';
+
+interface Props {
+  repo: Repo;
+}
+
+const cx = classNames.bind(styles);
+
+const RepoItem: React.FC<Props> = ({ repo }) => {
```

Comment:
repo의 속성이 여러개라면 다음과 같이 더 세세하게 쪼개서 사용도 가능합니다.

```suggestion
const RepoItem: React.FC<Props> = ({ repo: { name, star } }) => {

<p>{ name }</p>
```

또는

```typescript
const { name, star } = repo;
```

Reply:
오 그렇네요! 감사합니다